src/remotebuild/lib/darwin/darwinCerts.ts

```diff
@@ -28,11 +28,10 @@ import tacoUtils = require ("taco-utils");
 
 import utils = tacoUtils.UtilHelper;
 import logger = tacoUtils.Logger;
-var exec = child_process.exec;
 
 class Certs {
-    private static debug = false;
-    private static CERT_DEFAULTS = {
+    private static debug: boolean = false;
+    private static CERT_DEFAULTS: any = {
```

Comment:
Please put the proper type for this variable. With any, you can use any members, like dayss and the compiler will allow it. If you leave it untyped, typescript deduces the proper type, so trying to use Certs.CERT_DEFAULTS.dayss gives this error:
remotebuild\lib\darwin\darwinCerts.ts(227,64): error TS2339: Property 'dayss' does not exist on type '{ days: number; country: string; ca_cn: string; pfx_name: string; client_cn: string; }'.